README.md

```diff
@@ -145,6 +173,16 @@ sparkYarnSink.additionalConfs.spark.yarn.keytab=
 sparkYarnSink.additionalConfs.spark.yarn.principal=
 sparkYarnSink.additionalConfs.spark.shuffle.service.enabled=true
 sparkYarnSink.additionalConfs.spark.dynamicAllocation.enabled=true
+
+#Submit api = EMR
+spark.emr.clusterId=
+spark.emr.filesToDeploy=
+spark.emr.additionalConfs=
+
+#Common properties for Submit api = YARN and EMR
+sparkYarnSink.hadoopResourceManagerUrlBase=
+sparkYarnSink.userUsedToKillJob=
+spark.submit.thread.pool.size=10
 ```
 ```
 #Postgresql properties for connection to trigger database
```

Comment:
I would also add `spring.liquibase.change-log=classpath:/db_scripts/liquibase/db.changelog.yml`, otherwise the app doesn't work.

Reply:
Done